src/postings/mod.rs

```diff
@@ -173,30 +203,20 @@ pub mod tests {
         let reader = index.reader().unwrap();
         let mut index_writer = index.writer_for_tests().unwrap();
         index_writer.set_merge_policy(Box::new(NoMergePolicy));
-        {
-            index_writer.add_document(doc!(text_field=>exceeding_token_text))?;
-            index_writer.commit()?;
-            reader.reload()?;
-            let searcher = reader.searcher();
-            let segment_reader = searcher.segment_reader(0u32);
-            let inverted_index = segment_reader.inverted_index(text_field)?;
-            assert_eq!(inverted_index.terms().num_terms(), 1);
-            let mut bytes = vec![];
-            assert!(inverted_index.terms().ord_to_term(0, &mut bytes)?);
-            assert_eq!(&bytes, b"hello");
-        }
-        {
-            index_writer.add_document(doc!(text_field=>ok_token_text.clone()))?;
-            index_writer.commit()?;
-            reader.reload()?;
-            let searcher = reader.searcher();
-            let segment_reader = searcher.segment_reader(1u32);
```

Comment:
Since now the segment orders are only dependent on the document count, this test will be flaky.
Fixed to separate tests not to dependent the segment id order.